install/syndesis.yml

```diff
@@ -580,6 +580,20 @@ objects:
     resources:
       requests:
         storage: ${POSTGRESQL_VOLUME_CAPACITY}
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    name: syndesis-upgrade
+    labels:
+      app: syndesis
+      syndesis.io/app: syndesis
+      syndesis.io/type: infrastructure
+  spec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: ${POSTGRESQL_VOLUME_CAPACITY}
```

Comment:
Not sure whether we really should reuse the DB volume limit here as I expect that the upgrade pod PVC can be larger (holding multiple backups for a DB). While thinking about we probably should also some kind of cleanup mechanism to the update to e.g. retain only the last 3 copies (so then I would expect a 2-3 times as large capacity for the upgrade pod).

Would you mind to introduce a `UPGRADE_VOLUME_CAPACITY` template parameter here ? And ideally a corresponding entry in the Syndesis CRD.

If you don't have spare cycles for this, I could pick it up later but I think it would be helpful to stay consistent.

Reply:
Yes, it makes sense. I can do it.